shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -19,6 +19,10 @@
 static FlutterPlatformViewsTestMockPlatformView* gMockPlatformView = nil;
 const float kFloatCompareEpsilon = 0.001;
 
+@interface FlutterTouchInterceptingView (Test)
+- (id)accessibilityContainer;
```

Comment:
Please make a `_Test.h` file instead of re-declaring methods directly in test files.

Reply:
This entire redeclaration block should be removed in favor of including the test header.